src/main/kotlin/lotto/domain/LottoResult.kt

```diff
@@ -1,14 +1,13 @@
 package lotto.domain
 
 class LottoResult {
-    private val result = LottoPrize.values().associateWith { 0 }.toMutableMap()
+    private var result = LottoPrize.values().associateWith { 0 }.toMap()
```

Comment:
생성자 파라미터로 넣으면 var가 아니라 val이여도 가능할 것 같아요!
```suggestion
    LottoResult(private val result: EnumMap<LottoPrize, Int>)
```

Reply:
훨씬 간결해지고 보기 쉽네요!!